client/src/components/layout/TripTile.js

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import { Link } from "react-router-dom"
+
+const TripTile = (props) => {
+  let location = ""
+  if (props.trip.city) {
+    location = `${props.trip.city}, ${props.trip.country}`
+  } else {
+    location = `${props.trip.country}`
+  }
+
+  let numOfDays = ""
```

Comment:
For some of these properties that might not have a value, you might consider providing a default message if there is no value like an italicized "Not provided" so it's not just an empty string, there is some value there for the user (it can be confusing to users when they expect to see a value but none is there!)